app/model/command/UpdateTagCommand.scala

```diff
@@ -0,0 +1,20 @@
+package model.command
+
+import com.gu.tagmanagement.{EventType, TagEvent}
+import model.Tag
+import play.api.Logger
+import repositories.TagRepository
+import services.KinesisStreams
+
+
+case class UpdateTagCommand(tag: Tag) extends Command[Tag] {
+  override def process: Option[Tag] = {
+    Logger.info(s"updating tag ${tag.id}")
+
+    val result = TagRepository.upsertTag(tag)
```

Comment:
Does this mean the node gets the update twice? once here and once when it appears on the stream?